src/v/raft/consensus.cc

```diff
@@ -2021,16 +2039,9 @@ consensus::do_append_entries(append_entries_request&& r) {
           truncate_at);
         _probe->log_truncated();
 
-        // We are truncating the offset translator before truncating the log
-        // because if saving offset translator state fails, we will retry and
-        // eventually log and offset translator will become consistent. OTOH if
-        // log truncation were first and saving offset translator state failed,
-        // we wouldn't retry and log and offset translator could diverge.
```

Comment:
was this behavior / comment preserved?

Reply:
Yeah, this comment is in the log impl now